src/app/todoApp.js

```diff
@@ -0,0 +1,32 @@
+(function() {
```

Comment:
why do you need such syntax here?

Reply:
in past, we have to use IIFE for controllers, directives and services - so, as it is just a training and Lector said that we can do anything there - I implement that in such way